dotnet/samples/Demos/VectorStoreRAG/README.md

```diff
@@ -11,12 +11,15 @@ The sample can be configured in various ways:
    1. AzureAISearch
    1. AzureCosmosDBMongoDB
    1. AzureCosmosDBNoSQL
+   1. InMemory
```

Comment:
Should we add Volatile close to In-Memory for the legacy concept users?
```suggestion
   1. InMemory (Volatile)
```

Reply:
We are trying to deprecate the Volatile name, and Volatile still exists as obsolete code separate to InMemory, so I think adding it here might confuse people.